git-subtree-pull

```diff
@@ -13,7 +13,15 @@ for item; do
     esac
 done
 
-set -- "${POSITIONAL[@]}"
+FORWARD_ARGS=()
```

Comment:
@jstine35 how about this change? The issue is if we call for example `git-subtree-pull --add-if-missing nes https://github.com/implicitconversions/nes master` we will forward along the `--add-if-missing` and this will make `git-subtree` fail because this isn't an arg it supports.

Is there a better way I should be doing this in bash?

Reply:
That's supposed to get removed up top, eg. it shouldn't have made it into `POSITIONAL` and so it shouldn't be in the list of args that gets forwarded to the command.

I'd be interested to know how `--add-with-missing` is ending up in `POSITIONAL` and solve that problem instead.

Reply:
I found the bug.

https://github.com/implicitconversions/git-subtree-fast/blob/94d9427de0492869fc5ea3bdc37d73349315f524/git-subtree-pull#L8

`$lval` is supposed to be `$item`

Reply:
It also looks like `rval` isn't doing anything, so I assume that's also safe to delete?

Reply:
Yeah, I copy it in from a generic boilerplate since it's often useful for any assignment arguments, if/when we add them.

I find I often enough need it eventually that I just include it by default. But also, it takes 6 seconds to write when it's needed so I'm cool with it either being there or not being there.